test-suite/src/test/scala/org/scalajs/testsuite/javalib/CollectionTest.scala

```diff
@@ -165,7 +165,7 @@ trait CollectionTest extends JasmineTest with ExpectExceptions {
       coll.add("one")
       expect(coll.contains("one")).toBeTruthy
       expect(coll.contains("two")).toBeFalsy
-      expect(coll.contains(null)).toBeFalsy
+      //expect(coll.contains(null)).toBeFalsy
```

Comment:
Do not comment out code.

Either remove it, or execute this line conditionally depending on whether or not the CollectionFactory supports `null` values (better).